mailpoet/lib/PostEditorBlocks/WooCommerceBlocksIntegration.php

```diff
@@ -30,18 +33,23 @@ class WooCommerceBlocksIntegration {
   /** @var SubscribersRepository */
   private $subscribersRepository;
 
+  /** @var PluginHelper  */
+  private $pluginHelper;
+
   public function __construct(
     WPFunctions $wp,
     SettingsController $settings,
     WooCommerceSubscription $woocommerceSubscription,
     WooSegment $wooSegment,
-    SubscribersRepository $subscribersRepository
+    SubscribersRepository $subscribersRepository,
+    PluginHelper $pluginHelper
   ) {
     $this->wp = $wp;
     $this->settings = $settings;
     $this->woocommerceSubscription = $woocommerceSubscription;
     $this->wooSegment = $wooSegment;
     $this->subscribersRepository = $subscribersRepository;
+    $this->pluginHelper = $pluginHelper;
   }
 
   public function init() {
```

Comment:
I have tested your build, and the deprecation warning still occurs because we need to use a specific hook in the init method by the WC Block version. In my opinion, we should focus on hooks:
`woocommerce_blocks_checkout_update_order_from_request`
`woocommerce_store_api_checkout_update_order_from_request`
We know that the older hook `woocommerce_blocks_checkout_update_order_from_request` doesn't work in version 7.2.0, but I think that it will be solved by using the newer hook.
What do you think about it?

Reply:
Makes sense. Will do and take MAILPOET-4208 also into account for this PR.

Reply:
Addressed in [b0b73de](https://github.com/mailpoet/mailpoet/pull/4115/commits/b0b73dea16a4dbc3f253f665b75936d908b40a88). Thanks @lysyjan

Reply:
@websupporter I apologize, but warnings are still presented. I think the problem is bundled WC Blocks plugin in the WC plugin. You can see it in the installed WC plugin in the folder `packages`.  In my opinion, the plugin helper doesn't know about those plugins. I have found [another helper](https://github.com/mailpoet/mailpoet/blob/9f39843189084d2349d4bfbbaa175a3e45d2eac4/mailpoet/lib/WooCommerce/Helper.php#L10-L18). Maybe we can use this helper instead of new? I haven't checked how the WC plugin loads the WC Blocks plugin yet.